airbyte-integrations/connectors/source-mongodb-internal-poc/src/main/java/io/airbyte/integrations/source/mongodb/internal/cdc/MongoDbDebeziumPropertiesManager.java

```diff
@@ -0,0 +1,95 @@
+/*
+ * Copyright (c) 2023 Airbyte, Inc., all rights reserved.
+ */
+
+package io.airbyte.integrations.source.mongodb.internal.cdc;
+
+import static io.airbyte.integrations.source.mongodb.internal.MongoConstants.AUTH_SOURCE_CONFIGURATION_KEY;
+import static io.airbyte.integrations.source.mongodb.internal.MongoConstants.CONNECTION_STRING_CONFIGURATION_KEY;
+import static io.airbyte.integrations.source.mongodb.internal.MongoConstants.DATABASE_CONFIGURATION_KEY;
+import static io.airbyte.integrations.source.mongodb.internal.MongoConstants.PASSWORD_CONFIGURATION_KEY;
+import static io.airbyte.integrations.source.mongodb.internal.MongoConstants.USER_CONFIGURATION_KEY;
+
+import com.fasterxml.jackson.databind.JsonNode;
+import io.airbyte.integrations.debezium.internals.AirbyteFileOffsetBackingStore;
+import io.airbyte.integrations.debezium.internals.AirbyteSchemaHistoryStorage;
+import io.airbyte.integrations.debezium.internals.DebeziumPropertiesManager;
+import io.airbyte.protocol.models.v0.ConfiguredAirbyteCatalog;
+import io.airbyte.protocol.models.v0.ConfiguredAirbyteStream;
+import java.util.List;
+import java.util.Optional;
+import java.util.Properties;
+import java.util.stream.Collectors;
+
+/**
+ * Custom {@link DebeziumPropertiesManager} specific for the configuration of the Debezium MongoDB
+ * connector.
+ * <p />
+ * This implementation provides the specific connection properties required for the Debezium MongoDB
+ * connector. These properties differ from the general relational database connection properties
+ * used by the other Debezium connectors.
+ */
+public class MongoDbDebeziumPropertiesManager extends DebeziumPropertiesManager {
```

Comment:
Can we keep this class in Debezium package only.

Reply:
I will see what can be done with this.  The problem is the existing one is very relational-database heavy.  The Debezium MongoDB connector does not use the same property keys for configuration that the other relational database connectors use.  We will probably need to do some refactoring to support telling the debezium record publisher which one to use.  I am going to merge this as is and clean it up as I integration with Debezium, if that is okay with you @subodh1810 .